packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -244,8 +247,15 @@ hub.post("/", async (req, res) => {
           message: `Attempting to execute ${botName} serverless spoke using named spoke ${botConfig.spokeUrlName}`,
         });
       const spokeUrl = getSpokeUrl(botConfig.spokeUrlName);
+      const runId = botConfig.environmentVariables[RUN_IDENTIFIER_KEY];
       promiseArray.push(
-        Promise.race([_executeServerlessSpoke(spokeUrl, botConfig), _rejectAfterDelay(spokeRejectionTimeout, botName)])
+        Promise.race([
+          _executeServerlessSpoke(spokeUrl, botConfig, botName),
+          _rejectAfterDelay(spokeRejectionTimeout, botName),
```

Comment:
Should we add the `runId` to the `_rejectAfterDelay` or that's already present in the base logger?

Reply:
I'm sorry I don't quite understand the question. You're asking why it isn't passed into that function so it can be included in the error message?

Reply:
Yes, correct.